requirements/base.txt

```diff
@@ -5,43 +5,38 @@
 #    make upgrade
 #
 appdirs==1.4.3            # via fs
-backports.os==0.1.1       # via fs
 binaryornot==0.4.4        # via cookiecutter
-boto3==1.4.8              # via fs-s3fs
+boto3==1.12.9             # via fs-s3fs
 boto==2.39.0
-botocore==1.8.50          # via boto3, s3transfer
+botocore==1.15.9          # via boto3, s3transfer
 certifi==2019.11.28       # via requests
 chardet==3.0.4            # via binaryornot, requests
 cookiecutter==0.9.0
 django-pyfs==2.0
-django==1.11.26
+django==2.2.10
```

Comment:
@kaizoku Was this intended? django 2.2.10 isn't supported on python2. Discovered by a user on this forum thread: https://discuss.openedx.org/t/xblock-django-installation-failiure/1802

Reply:
@swalladge, not really, no. This PR was to update requirements so they would work for Django 2.2 and Python 3. There was a previous PR (https://github.com/edx/xblock-sdk/pull/165/files) for Django 2.2 support, but requirements wouldn't compile successfully still, which is why I opened this PR. I see that README.md still says Python 2.7 though, which is misleading now.

Unfortunately [`pip-compile` completely ignores `python_version` directives](https://github.com/jazzband/pip-tools/issues/585#issuecomment-381704945), and they've stated they won't support them, so there isn't really a good way to have different django requirements for different python versions as long as we're using pip-tools.

Should we add extra documentation to note which versions of xblock-utils are Django 1.11 compatible vs Django 2.2?

Reply:
> Should we add extra documentation to note which versions of xblock-utils are Django 1.11 compatible vs Django 2.2?

Probably yes. Currently, python2 support has been unexpectedly dropped with this PR. It sounds like formally dropping python2 is going to happen soon anyway; it's just happened sooner than expected. :P